_data/retail.yml

```diff
@@ -53,8 +53,10 @@ websites:
   - name: Best Buy
     url: https://www.bestbuy.com/
     img: bestbuy.png
-    twitter: BestBuySupport
-    facebook: bestbuy
+    tfa:
+      - sms
+      - software
+    doc: https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c?id=pcmcat1561056149844
```

Comment:
```suggestion
    doc: https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c
```

Reply:
While both the URLs, with and without the id query param, result in the same page being displayed, the canonical link is with the id query param.  The canonical URL is the preferred version. 

<link rel="canonical" href="https://www.bestbuy.com/site/help-topics/2-step-verification/pcmcat1561056149844.c?id=pcmcat1561056149844">
https://en.wikipedia.org/wiki/Canonical_link_element

Reply:
We want to keep the URLs as short as possible so please remove any part of the URL that's not required for the page to load.